raiden/api/python.py

```diff
@@ -1318,5 +1324,13 @@ def withdraw_from_udc(self, amount: TokenAmount) -> TransactionHash:
             amount=amount, given_block_identifier=confirmed_block_identifier
         )
 
+    def get_new_notifications(self):
+        notifications = self.raiden.notifications
+        self.raiden.notifications = defaultdict(lambda x: None)
```

Comment:
Do we need a defaultdict, or would a plain dict work just as well?

Reply:
You're right, the defaultdict is not needed ✅